### PR TITLE
Update dependencies

### LINKED ISSUE #141: Update module versions
### Type of Issue

- [ ] Bug
- [ ] Feature Request
- [ ] Documentation
- [X] Other

### Description

Well, I think dependabot don't like us. I'll do my job.

### PR DESCRIPTION
**Pull request by KwikKill**
## Description

Update ALL dependencies to latest version.
This can cause unexpected problems about anywhere so complete reviews are needed.

## Checklist

- [X] I have tested the changes locally and they work as expected.
- [X] I have tested the responsiveness of the changes and they work as expected.
- [X] I have assigned the pull request to the appropriate reviewer(s).
- [X] I have added labels to the pull request, if necessary.

## Related Issues

Fix #141

### CODE REVIEW

**Review by Dyskal — APPROVED** [review]
LGTM

File: src/views/Me.vue [hunk 1]
```diff
@@ -657,17 +660,17 @@ const openScoreModal = () => {
...
         >
-          <label :for="'input' + n">Partie {{ n }} </label>
+          <label :for="`input${n}`">Partie {{ n }} </label>
           <input
-            :id="'input' + n"
+            :id="`input${n}`"
             v-model="data_score.times[n]"
             :class="{ error: context.invalid }"
-            aria-label="duration"
+            aria-label="duration (en minutes)"
```
Comment (istar410) [inline src/views/Me.vue:671]: There is a mix of English and French.

**Review by istar410 — APPROVED** [review]
LGTM, there is just a tiny typo.

**Review by floflo0 — APPROVED** [review]
LGTM

**Pull request merged**